system_reports/linux-docker-android.log

```diff
@@ -10,54 +10,57 @@
 
 
 === Revision / ID ======================
-* BITRISE_DOCKER_REV_NUMBER_BASE: v2018_06_14_1
+* BITRISE_DOCKER_REV_NUMBER_BASE: v2018_06_28_1
 ========================================
 
 
 === Pre-installed lib versions ========
-* zlib1g-dev: 1:1.2.8.dfsg-2ubuntu4.1
-* libssl-dev: 1.0.2g-1ubuntu4.12
-* libreadline6-dev: 6.3-8ubuntu2
-* libyaml-dev: 0.1.6-3
-* libsqlite3-dev: 3.11.0-1ubuntu1
+* zlib1g-dev: 1:1.2.11.dfsg-0ubuntu2
+* libssl-dev: 1.1.0g-2ubuntu4.1
+dpkg-query: package 'libreadline6-dev' is not installed and no information is available
```

Comment:
Should we remove this lib version report @tamaseger81 ? Or adjust to a newer version?